src/main/java/io/zold/api/Taxes.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2018 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package io.zold.api;
+
+import org.cactoos.Proc;
+import org.cactoos.text.FormattedText;
+
+/**
+ * Taxes payment algorithm.
+ *
+ * @since 1.0
+ * @todo #40:30min Implement tax payment to remote nodes.
+ *  Payment should happen only if the wallet is in debt of more than
+ *  1 Zold. Debt is difference between current taxes that should have
+ *  been paid by wallet (see whitepaper for formula) and how much it
+ *  already paid in the past. A first algorithm could pay the
+ *  max to each node until there is nothing else to pay. The test
+ *  must also be updated.
+ */
+public final class Taxes implements Proc<Wallet> {
+
+    /**
+     * The beneficiary nodes.
+     */
+    private final Iterable<Remote> bnfs;
+
+    /**
+     * Ctor.
+     *
+     * @param nodes Remote nodes.
+     */
+    public Taxes(final Iterable<Remote> nodes) {
+        this.bnfs = new TaxBeneficiaries(nodes);
+    }
+
+    @Override
+    public void exec(final Wallet wallet) throws Exception {
+        throw new UnsupportedOperationException(
+            new FormattedText(
```

Comment:
@carlosmiranda this is not meant to stay like this, this will be implemented and thus at that time, we will avoid throwing exception if it makes sense.

Reply:
@victornoel let's do it. `UncheckedText` will certainly be removed upon implementation but implementor may forget to remove/narrow down declared `Exception`. Makes sense?

Reply:
@carlosmiranda ok, good for me